src/qt/signverifymessagedialog.cpp

```diff
@@ -10,6 +10,7 @@
 #include <qt/platformstyle.h>
 #include <qt/walletmodel.h>
 
+#include <config/bitcoin-config.h>
```

Comment:
It is https://github.com/bitcoin/bitcoin/pull/29494 goal to include this headers unguarded :)

Effectively, this PR conflicts with it. However, I believe it will be merged soon.

Anyway, it seems reasonable in this PR to use:
```c++
#if defined(HAVE_CONFIG_H)
#include <config/bitcoin-config.h>
#endif
```
or
```c++
#include <config/bitcoin-config.h> // IWYU pragma: keep
```

Reply:
Thanks hebasto, I hadn't seen that in progress. Went for the latter suggestion in fb9f150759b22772dd48983a2be1ea397245e289 as it seems to be preferred by marco in the other PR.